client/js/test/table-view.test.js

```diff
@@ -12,6 +12,28 @@ describe('table-view', () => {
 
 	});
 
+	describe('footer sums', () => {
```

Comment:
Add more tests that test your new feature functionality. You should develop some code, then write one or more tests for it, then develop more code, then write and run one or more tests for it. That way you can't possibly reach a state of terrible debugging.